.changeset/breezy-monkeys-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"alephium-desktop-wallet": patch
+---
+
+New NFT details modal + simplify NFT cards + redesign transaction details modal
```

Comment:
nit: It would help with creating a changelog if this was split into different files ✨ Is this the format you'd like to see it in the changelog? :/

Reply:
😢